libpod/runtime_ctr.go

```diff
@@ -579,6 +579,17 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force bool,
 
 	c.newContainerEvent(events.Remove)
 
+	for _, v := range c.config.ImageVolumes {
```

Comment:
This should be part of container cleanup, not container removal. We want to unmount these when the container stops.